dap-etl/transformation/src/main/scala/org/broadinstitute/monster/dap/cslb/CslbTransformationPipelineBuilder.scala

```diff
@@ -41,19 +41,21 @@ object CslbTransformationPipelineBuilder extends PipelineBuilder[Args] {
         s"${args.inputPrefix}/records/*.json"
       )
 
-    // Group by study ID (record number) and field name
-    // to get the format: (studyId, Iterable((fieldName, Iterable(value))))
+    // Group by study ID (record number) and arm (address_year_month)
```

Comment:
⛏️ This may be a copy/paste typo from the env pipeline, I don't think the CSLB arms represent `address_year_month`

Reply:
you are correct, fixing this now